chromium_src/components/security_interstitials/content/https_only_mode_blocking_page.cc

```diff
@@ -0,0 +1,22 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "components/security_interstitials/content/https_only_mode_blocking_page.h"
+
+namespace security_interstitials {
+
+namespace {
+const char kBraveLearnMoreLink[] =
+    "https://support.brave.com/hc/en-us/articles/9809690466061";
+}  // namespace
+
+}  // namespace security_interstitials
+
+#define OpenUrlInNewForegroundTab(...) \
+  OpenUrlInNewForegroundTab(GURL(kBraveLearnMoreLink))
```

Comment:
We might want to `#include "components/security_interstitials/content/security_interstitial_controller_client.h", so that we don't redefine the function in the header.

Reply:
Good point -- fixed.